lua/lvim/lsp/init.lua

```diff
@@ -144,7 +151,13 @@ function M.setup()
     return
   end
 
+  local isNvim06 = vim.fn.has "nvim-0.5.1" > 0
+
   for _, sign in ipairs(lvim.lsp.diagnostics.signs.values) do
+    local nvim06SignName = NVIM_05_TO_06_SIGN_NAMES[sign.name]
+    if isNvim06 and nvim06SignName then
+      vim.fn.sign_define(nvim06SignName, { texthl = nvim06SignName, text = sign.text, numhl = nvim06SignName })
+    end
     vim.fn.sign_define(sign.name, { texthl = sign.name, text = sign.text, numhl = sign.name })
   end
```

Comment:
```suggestion
  local is_neovim_nightly = vim.fn.has "nvim-0.5.1" > 0

  for _, sign in ipairs(lvim.lsp.diagnostics.signs.values) do
    local  lsp_sign_name = LSP_SIGN_MAP[sign.name]
    if is_neovim_nightly and lsp_sign_name then
      vim.fn.sign_define(lsp_sign_name, { texthl = lsp_sign_name, text = sign.text, numhl = lsp_sign_name })
    else
      vim.fn.sign_define(sign.name, { texthl = sign.name, text = sign.text, numhl = sign.name })
    end
  end
```

Reply:
Don't we need the `else` here?

Reply:
No, the deprecated names for signs still need to be defined for plugins that haven't been updated yet.